### PR TITLE
Update UG: FAQ, Known Issues and Command Summary

### PR DESCRIPTION
**Pull request by ruo-x**
Perhaps we can come up with a standardized format for check, add, remove volunteers for easier parsing of inputs.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
...
+| **List all volunteers**                       | `vlist`                                                                                                                                                                                                                                                                                                                |
+| **Delete a volunteer profile**                | `vdelete VOLUNTEER_ID` <br> e.g., `vdelete 4`                                                                                                                                                                                                                                                                          |
+| **Add a volunteer to an event**               | `vadd vn/VOLUNTEER hp/PHONE_NUMBER e/EMAIL /to eid/EVENT_ID`<br> e.g., `vadd vn/John hp/81234567 e/john123@gmail.com /to eid/3` <br> Alternatively, `vadd vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL /to en/EVENT_NAME` <br> e.g., `vadd vn/John hp/81234567 e/john123@gmail.com /to en/Fundraising for needy families` |
```
Comment (jordankanghm) [inline docs/UserGuide.md:239]: i think the command format has changed, can see max's pull request

**Review by jordankanghm — CHANGES_REQUESTED** [review]
Just a small issue with vadd

File: docs/UserGuide.md [hunk 2]
```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
...
+| **Read an individual event**                  | `eshow EVENT_ID` <br> e.g., `eshow 8`                                                                                                                                                                                                                                                                                  |
+| **Delete an event**                           | `edelete EVENT_ID` <br> e.g., `edelete 3`                                                                                                                                                                                                                                                                              |
+| **Create a new volunteer profile**            | `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com`                                                                                                                                                                                                     |
```
Comment (brein62) [inline docs/UserGuide.md:236]: can update `vcreate` here to reflect my new `skills` parameter I just added:
```suggestion
| **Create a new volunteer profile**            | `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL [s/SKILLS]`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                                                                                   
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
...
+| **List all volunteers**                       | `vlist`                                                                                                                                                                                                                                                                                                                |
```
Comment (brein62) [inline docs/UserGuide.md:237]: okays thanks for reminding me to add `vlist` LOL

can also add `vshow`:
```md
| **Read an individual volunteer's details**                  | `vshow VOLUNTEER_ID` <br> e.g., `vshow 6`                                                                                                                                                                                                                                                                               |
```

**Review by brein62 — CHANGES_REQUESTED** [review]
overall LGTM! just need a few changes wrt my part in the UG 🥵

File: docs/UserGuide.md (hunk 1 above)
Comment (ruo-x) [reply docs/UserGuide.md:239]: oki

File: docs/UserGuide.md [hunk 4]
```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
...
+| **Add a volunteer to an event**               | `eaddv vid/VOLUNTEER_ID /to eid/EVENT_ID`<br> e.g., `eaddv vid/1 /to eid/3` <br> Alternatively, `eaddv vn/VOLUNTEER_NAME /to en/EVENT_NAME` <br> e.g., `eaddv vn/Betsy Crowe /to en/Fundraising for needy families`                                                                    |
+| **Check for volunteers assigned to an event** | `elistv eid/EVENT_ID` <br> e.g. `elistv eid/8` <br> Alternatively, `elistv en/EVENT_NAME` <br> e.g., `elistv en/October beach clean up`                                                                                                                                                |
+| **Remove a volunteer from an event**          | `eremovev vid/VOLUNTEER_ID /from eid/EVENT_ID`<br> e.g., `eremovev vid/3 /from eid/4` <br> Alternatively, `eremovev vn/VOLUNTEER_NAME /from en/EVENT_NAME` <br> e.g., `eremovev vn/John Lim /from en/October beach clean up`                                                           |
```
Comment (tiongMax) [inline docs/UserGuide.md:241]: Based on what we discussed earlier, i think the /from can be removed.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
...
+| **List all volunteer profiles**               | `vlist`                                                                                                                                                                                                                                                                                |
+| **Delete a volunteer profile**                | `vdelete VOLUNTEER_ID` <br> e.g., `vdelete 4`                                                                                                                                                                                                                                          |
+| **Add a volunteer to an event**               | `eaddv vid/VOLUNTEER_ID /to eid/EVENT_ID`<br> e.g., `eaddv vid/1 /to eid/3` <br> Alternatively, `eaddv vn/VOLUNTEER_NAME /to en/EVENT_NAME` <br> e.g., `eaddv vn/Betsy Crowe /to en/Fundraising for needy families`                                                                    |
```
Comment (tiongMax) [inline docs/UserGuide.md:239]: Based on what we discussed earlier, i think the /to can be removed.

**Review by tiongMax — CHANGES_REQUESTED** [review]
Overall lgtm, just some minor details need to be updated and it is understandable since we are updating the command format frequently.

File: docs/UserGuide.md (hunk 4 above)
Comment (ruo-x) [reply docs/UserGuide.md:241]: okai will make the necessary changes

File: docs/UserGuide.md (hunk 5 above)
Comment (ruo-x) [reply docs/UserGuide.md]: dones

**Review by jordankanghm — APPROVED** [review]
lgtm!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
...
+| **List all events**                           | `elist`                                                                                                                                                                                                                                                                                |
+| **Read an individual event**                  | `eshow EVENT_ID` <br> e.g., `eshow 8`                                                                                                                                                                                                                                                  |
+| **Delete an event**                           | `edelete EVENT_ID` <br> e.g., `edelete 3`                                                                                                                                                                                                                                              |
```
Comment (brein62) [inline docs/UserGuide.md:235]: to confirm with zac

**Review by brein62 — CHANGES_REQUESTED** [review]
overall LGTM, but can add `vclear` and `eclear` features in the future (analogous to the `clear` feature in AB3)

**Review by brein62 — APPROVED** [review]

**Comment by brein62** [comment]
> Overall lgtm, just some minor details need to be updated and it is understandable considering how often we change the format

@tiongMax dont forget to approve if the changes are to ur liking 👍

**Comment by tiongMax** [comment]
> > Overall lgtm, just some minor details need to be updated and it is understandable considering how often we change the format
> 
> @tiongMax dont forget to approve if the changes are to ur liking 👍

What if it is not?

**Review by tiongMax — APPROVED** [review]
lgtm, steady

**Pull request merged**